manifests/i/iniBuilds/iniManager/2.3.5/iniBuilds.iniManager.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: iniBuilds.iniManager
+PackageVersion: 2.3.5
+InstallerType: exe
+InstallModes:
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-04-19
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
This installed normally in a 64-bit Windows 10 VM, so the only guess I have is that it might not work on 32-bit OSes.